components/gantry/oneaxis/oneaxis_test.go

```diff
@@ -10,86 +11,49 @@ import (
 	"go.viam.com/test"
 
 	"go.viam.com/rdk/components/board"
-	fakeencoder "go.viam.com/rdk/components/encoder/fake"
 	"go.viam.com/rdk/components/gantry"
 	"go.viam.com/rdk/components/motor"
-	"go.viam.com/rdk/components/motor/fake"
 	"go.viam.com/rdk/referenceframe"
 	"go.viam.com/rdk/resource"
 	"go.viam.com/rdk/testutils/inject"
 )
 
-const motorName = "x"
-
-func createFakeMotor() *inject.Motor {
-	fakeMotor := &inject.Motor{}
-
-	fakeMotor.PropertiesFunc = func(ctx context.Context, extra map[string]interface{}) (map[motor.Feature]bool, error) {
-		return map[motor.Feature]bool{
-			motor.PositionReporting: true,
-		}, nil
-	}
-
-	fakeMotor.PositionFunc = func(ctx context.Context, extra map[string]interface{}) (float64, error) { return 1, nil }
+const (
+	motorName = "x"
+	testGName = "test"
+	boardName = "board"
+)
 
-	fakeMotor.ResetZeroPositionFunc = func(ctx context.Context, offset float64, extra map[string]interface{}) error { return nil }
+var count = 0
 
-	fakeMotor.GoToFunc = func(ctx context.Context, rpm, position float64, extra map[string]interface{}) error {
-		return nil
-	}
-	fakeMotor.GoForFunc = func(ctx context.Context, rpm, revolutions float64, extra map[string]interface{}) error {
-		return nil
+func createFakeMotor() motor.Motor {
+	return &inject.Motor{
+		PropertiesFunc: func(ctx context.Context, extra map[string]interface{}) (map[motor.Feature]bool, error) {
+			return map[motor.Feature]bool{motor.PositionReporting: true}, nil
+		},
+		PositionFunc: func(ctx context.Context, extra map[string]interface{}) (float64, error) {
+			return float64(count + 1), nil
+		}, ResetZeroPositionFunc: func(ctx context.Context, offset float64, extra map[string]interface{}) error { return nil },
```

Comment:
[nit] extremely minor, but new line before `ResetZeroPositionFunc` for consistency/it would look nice